tests/brokers/nats/test_kv_declarer_cache.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from faststream.nats import NatsBroker
+
+
+@pytest.mark.asyncio()
+async def test_object_storage_cache():
+    broker = NatsBroker()
+    await broker.connect()
+    bucket = await broker.key_value("test")
```

Comment:
We should check, that this method does not creates object twice. Please, patch `nats-py.create_key_value` method and check that is called once if `broker.key_value` calling multiple times